### PR TITLE
Add kt60-MU PCB

### PR DESCRIPTION
**Pull request by key10iq**
## Description
kt60-MU is a 60% PCB supporting QMK and VIA, compatible with many 60% keyboards

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added new keyboard - keyten/kt60_mu (QMK and VIA)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keyten/kt60_mu/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/keyten/kt60_mu/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/keyten/kt60_mu/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/keyten/kt60_mu/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/keyten/kt60_mu/kt60_mu.c [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+#include "quantum.h"
+
+#define LED_INDICATOR_PIN B11
+
+void matrix_init_kb(void) {
+    gpio_set_pin_output(LED_INDICATOR_PIN);
+    gpio_write_pin_high(LED_INDICATOR_PIN);
+    matrix_init_user();
+}
```
Comment (waffle87) [inline keyboards/keyten/kt60_mu/kt60_mu.c:12]: Should fix lint error.
```suggestion
// Copyright 2024 QMK Collaborators (@qmk)
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"

#define LED_INDICATOR_PIN B11

void matrix_init_kb(void) {
    gpio_set_pin_output(LED_INDICATOR_PIN);
    gpio_write_pin_high(LED_INDICATOR_PIN);
    matrix_init_user();
}
```

Comment (key10iq) [reply keyboards/keyten/kt60_mu/kt60_mu.c:12]: Hmm
image

Comment (sigprof) [reply keyboards/keyten/kt60_mu/kt60_mu.c:12]:
```
❯ file keyboards/keyten/kt60_mu/kt60_mu.c
keyboards/keyten/kt60_mu/kt60_mu.c: C source, ASCII text, with CRLF line terminators
```
You need to use LF line terminators in the repo.

Comment (key10iq) [reply keyboards/keyten/kt60_mu/kt60_mu.c:12]: I can't do this using GitHub web editor...

**Review by waffle87 — APPROVED** [review]

File: keyboards/keyten/kt60_mu/readme.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* Standard left position
+* Central position like Aksara 60
+
+Supports three JST connector position like the following PCBs or keyboards:
+
+* hiney h60
+* Singa Unikorn
+* Polaris V2
+
+![kt60-MU](https://i.imgur.com/72uSWLx.jpg)
```
Comment (zvecr) [inline keyboards/keyten/kt60_mu/readme.md:20]: The image here shows a the board supports both 2u and split backspace. We would expect more than the single layout macro defined in `keyboard.json`.

File: keyboards/keyten/kt60_mu/kt60_mu.c [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
```
Comment (zvecr) [inline keyboards/keyten/kt60_mu/kt60_mu.c:12]: Whats the value add here? Its not toggled by anything.

File: keyboards/keyten/kt60_mu/readme.md (hunk 4 above)
Comment (key10iq) [reply keyboards/keyten/kt60_mu/readme.md:20]: I didn't find anything like that in community layouts, so I called it `LAYOUT_60_ansi_tsangan_split_rshift`

File: keyboards/keyten/kt60_mu/kt60_mu.c (hunk 5 above)
Comment (key10iq) [reply keyboards/keyten/kt60_mu/kt60_mu.c:12]: This is just needed to keep the LED on continuously

**Comment by key10iq** [comment]
@zvecr sorry for the ping, but...

File: keyboards/keyten/kt60_mu/kt60_mu.c (hunk 5 above)
Comment (drashna) [reply keyboards/keyten/kt60_mu/kt60_mu.c:12]: Wouldn't it make sense to set it as caps lock or some such?

**Comment by drashna** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

File: keyboards/keyten/kt60_mu/keyboard.json [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["B2", "B1", "B7", "B6", "B5"],
+        "cols": ["A15", "A4", "F1", "A5", "F0", "A6", "C15", "A7", "C14", "B0", "C13", "B10", "B4", "B3"]
+    },
+    "indicators": {
+        "caps_lock": "A14"
+    }
+    "community_layouts": ["60_tsangan_hhkb"],
```
Comment (dunk2k) [inline keyboards/keyten/kt60_mu/keyboard.json:28]:
```suggestion
    "community_layouts": ["60_tsangan_hhkb", "60_ansi_wkl_split_bs_rshift", "60_hhkb"],
```

File: keyboards/keyten/kt60_mu/keyboard.json [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+                { "matrix": [3, 13], "x": 14, "y": 3 },
+                { "matrix": [4, 0], "w": 1.5, "x": 0, "y": 4 },
+                { "matrix": [4, 1], "x": 1.5, "y": 4 },
+                { "matrix": [4, 2], "w": 1.5, "x": 2.5, "y": 4 },
+                { "matrix": [4, 6], "w": 7, "x": 4, "y": 4 },
+                { "matrix": [4, 10], "w": 1.5, "x": 11, "y": 4 },
+                { "matrix": [4, 11], "x": 12.5, "y": 4 },
+                { "matrix": [4, 13], "w": 1.5, "x": 13.5, "y": 4 }
+            ]
+        }
```
Comment (dunk2k) [inline keyboards/keyten/kt60_mu/keyboard.json:160]:
```suggestion
        },
        "LAYOUT_60_ansi_wkl_split_bs_rshift": {
            "layout": [
                { "matrix": [0, 0], "x": 0, "y": 0 },
                { "matrix": [0, 1], "x": 1, "y": 0 },
                { "matrix": [0, 2], "x": 2, "y": 0 },
                { "matrix": [0, 3], "x": 3, "y": 0 },
                { "matrix": [0, 4], "x": 4, "y": 0 },
                { "matrix": [0, 5], "x": 5, "y": 0 },
                { "matrix": [0, 6], "x": 6, "y": 0 },
                { "matrix": [0, 7], "x": 7, "y": 0 },
                { "matrix": [0, 8], "x": 8, "y": 0 },
                { "matrix": [0, 9], "x": 9, "y": 0 },
                { "matrix": [0, 10], "x": 10, "y": 0 },
                { "matrix": [0, 11], "x": 11, "y": 0 },
                { "matrix": [0, 12], "x": 12, "y": 0 },
                { "matrix": [0, 13], "x": 13, "y": 0 },
                { "matrix": [2, 13], "x": 14, "y": 0 },
                { "matrix": [1, 0], "w": 1.5, "x": 0, "y": 1 },
                { "matrix": [1, 1], "x": 1.5, "y": 1 },
                { "matrix": [1, 2], "x": 2.5, "y": 1 },
                { "matrix": [1, 3], "x": 3.5, "y": 1 },
                { "matrix": [1, 4], "x": 4.5, "y": 1 },
                { "matrix": [1, 5], "x": 5.5, "y": 1 },
                { "matrix": [1, 6], "x": 6.5, "y": 1 },
                { "matrix": [1, 7], "x": 7.5, "y": 1 },
                { "matrix": [1, 8], "x": 8.5, "y": 1 },
                { "matrix": [1, 9], "x": 9.5, "y": 1 },
                { "matrix": [1, 10], "x": 10.5, "y": 1 },
                { "matrix": [1, 11], "x": 11.5, "y": 1 },
                { "matrix": [1, 12], "x": 12.5, "y": 1 },
                { "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
                { "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
                { "matrix": [2, 1], "x": 1.75, "y": 2 },
                { "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "matrix": [2, 3], "x": 3.75, "y": 2 },
                { "matrix": [2, 4], "x": 4.75, "y": 2 },
                { "matrix": [2, 5], "x": 5.75, "y": 2 },
                { "matrix": [2, 6], "x": 6.75, "y": 2 },
                { "matrix": [2, 7], "x": 7.75, "y": 2 },
                { "matrix": [2, 8], "x": 8.75, "y": 2 },
                { "matrix": [2, 9], "x": 9.75, "y": 2 },
                { "matrix": [2, 10], "x": 10.75, "y": 2 },
                { "matrix": [2, 11], "x": 11.75, "y": 2 },
                { "matrix": [2, 12], "w": 2.25, "x": 12.75, "y": 2 },
                { "matrix": [3, 0], "w": 2.25, "x": 0, "y": 3 },
                { "matrix": [3, 1], "x": 2.25, "y": 3 },
                { "matrix": [3, 2], "x": 3.25, "y": 3 },
                { "matrix": [3, 3], "x": 4.25, "y": 3 },
                { "matrix": [3, 4], "x": 5.25, "y": 3 },
                { "matrix": [3, 5], "x": 6.25, "y": 3 },
                { "matrix": [3, 6], "x": 7.25, "y": 3 },
                { "matrix": [3, 7], "x": 8.25, "y": 3 },
                { "matrix": [3, 8], "x": 9.25, "y": 3 },
                { "matrix": [3, 9], "x": 10.25, "y": 3 },
                { "matrix": [3, 10], "x": 11.25, "y": 3 },
                { "matrix": [4, 12], "w": 1.75, "x": 12.25, "y": 3 },
                { "matrix": [3, 13], "x": 14, "y": 3 },
                { "matrix": [4, 0], "w": 1.5, "x": 0, "y": 4 },
                { "matrix": [4, 2], "w": 1.5, "x": 2.5, "y": 4 },
                { "matrix": [4, 6], "w": 7, "x": 4, "y": 4 },
                { "matrix": [4, 10], "w": 1.5, "x": 11, "y": 4 },
                { "matrix": [4, 13], "w": 1.5, "x": 13.5, "y": 4 }
            ]
        },
        "LAYOUT_60_hhkb": {
            "layout": [
                { "matrix": [0, 0], "x": 0, "y": 0 },
                { "matrix": [0, 1], "x": 1, "y": 0 },
                { "matrix": [0, 2], "x": 2, "y": 0 },
                { "matrix": [0, 3], "x": 3, "y": 0 },
                { "matrix": [0, 4], "x": 4, "y": 0 },
                { "matrix": [0, 5], "x": 5, "y": 0 },
                { "matrix": [0, 6], "x": 6, "y": 0 },
                { "matrix": [0, 7], "x": 7, "y": 0 },
                { "matrix": [0, 8], "x": 8, "y": 0 },
                { "matrix": [0, 9], "x": 9, "y": 0 },
                { "matrix": [0, 10], "x": 10, "y": 0 },
                { "matrix": [0, 11], "x": 11, "y": 0 },
                { "matrix": [0, 12], "x": 12, "y": 0 },
                { "matrix": [0, 13], "x": 13, "y": 0 },
                { "matrix": [2, 13], "x": 14, "y": 0 },
                { "matrix": [1, 0], "w": 1.5, "x": 0, "y": 1 },
                { "matrix": [1, 1], "x": 1.5, "y": 1 },
                { "matrix": [1, 2], "x": 2.5, "y": 1 },
                { "matrix": [1, 3], "x": 3.5, "y": 1 },
                { "matrix": [1, 4], "x": 4.5, "y": 1 },
                { "matrix": [1, 5], "x": 5.5, "y": 1 },
                { "matrix": [1, 6], "x": 6.5, "y": 1 },
                { "matrix": [1, 7], "x": 7.5, "y": 1 },
                { "matrix": [1, 8], "x": 8.5, "y": 1 },
                { "matrix": [1, 9], "x": 9.5, "y": 1 },
                { "matrix": [1, 10], "x": 10.5, "y": 1 },
                { "matrix": [1, 11], "x": 11.5, "y": 1 },
                { "matrix": [1, 12], "x": 12.5, "y": 1 },
                { "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
                { "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
                { "matrix": [2, 1], "x": 1.75, "y": 2 },
                { "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "matrix": [2, 3], "x": 3.75, "y": 2 },
                { "matrix": [2, 4], "x": 4.75, "y": 2 },
                { "matrix": [2, 5], "x": 5.75, "y": 2 },
                { "matrix": [2, 6], "x": 6.75, "y": 2 },
                { "matrix": [2, 7], "x": 7.75, "y": 2 },
                { "matrix": [2, 8], "x": 8.75, "y": 2 },
                { "matrix": [2, 9], "x": 9.75, "y": 2 },
                { "matrix": [2, 10], "x": 10.75, "y": 2 },
                { "matrix": [2, 11], "x": 11.75, "y": 2 },
                { "matrix": [2, 12], "w": 2.25, "x": 12.75, "y": 2 },
                { "matrix": [3, 0], "w": 2.25, "x": 0, "y": 3 },
                { "matrix": [3, 1], "x": 2.25, "y": 3 },
                { "matrix": [3, 2], "x": 3.25, "y": 3 },
                { "matrix": [3, 3], "x": 4.25, "y": 3 },
                { "matrix": [3, 4], "x": 5.25, "y": 3 },
                { "matrix": [3, 5], "x": 6.25, "y": 3 },
                { "matrix": [3, 6], "x": 7.25, "y": 3 },
                { "matrix": [3, 7], "x": 8.25, "y": 3 },
                { "matrix": [3, 8], "x": 9.25, "y": 3 },
                { "matrix": [3, 9], "x": 10.25, "y": 3 },
                { "matrix": [3, 10], "x": 11.25, "y": 3 },
                { "matrix": [4, 12], "w": 1.75, "x": 12.25, "y": 3 },
                { "matrix": [3, 13], "x": 14, "y": 3 },
                { "matrix": [4, 1], "x": 1.5, "y": 4 },
                { "matrix": [4, 2], "w": 1.5, "x": 2.5, "y": 4 },
                { "matrix": [4, 6], "w": 7, "x": 4, "y": 4 },
                { "matrix": [4, 10], "w": 1.5, "x": 11, "y": 4 },
                { "matrix": [4, 11], "x": 12.5, "y": 4 }
            ]
        }
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
Community Layout support addition

File: keyboards/keyten/kt60_mu/keyboard.json [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+    "matrix_pins": {
+        "rows": ["B2", "B1", "B7", "B6", "B5"],
+        "cols": ["A15", "A4", "F1", "A5", "F0", "A6", "C15", "A7", "C14", "B0", "C13", "B10", "B4", "B3"]
+    },
+    "indicators": {
+        "caps_lock": "A14"
+    }
+    "community_layouts": ["60_tsangan_hhkb", "60_ansi_wkl_split_bs_rshift", "60_hhkb"],
+    "layouts": {
+        "LAYOUT_60_tsangan_hhkb": {
```
Comment (dunk2k) [inline keyboards/keyten/kt60_mu/keyboard.json:30]:
```suggestion
    "community_layouts": ["60_ansi_tsangan_split_bs_rshift", "60_ansi_wkl_split_bs_rshift", "60_hhkb"],
    "layouts": {
        "LAYOUT_60_ansi_tsangan_split_bs_rshift": {
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/keyten/kt60_mu/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT_60_tsangan_hhkb(
```
Comment (dunk2k) [inline keyboards/keyten/kt60_mu/keymaps/default/keymap.c:8]:
```suggestion
    [0] = LAYOUT_60_ansi_tsangan_split_bs_rshift(
```

File: keyboards/keyten/kt60_mu/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_RALT, KC_RGUI, KC_RCTL
+    ),
+
+    [1] = LAYOUT_60_tsangan_hhkb(
```
Comment (dunk2k) [inline keyboards/keyten/kt60_mu/keymaps/default/keymap.c:16]:
```suggestion
    [1] = LAYOUT_60_ansi_tsangan_split_bs_rshift(
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
CI build fix(es)

**Comment by key10iq** [comment]
Well...

**Pull request closed without merging**